tensorflow/python/kernel_tests/decode_raw_op_test.py

```diff
@@ -54,18 +54,26 @@ def testToInt16(self):
       self.assertEqual([None, None], decode.get_shape().as_list())
 
       result = decode.eval(feed_dict={in_bytes: ["AaBC"]})
-      if sys.byteorder == "big":
-        self.assertAllEqual(
-            [[ord("A") * 256 + ord("a"), ord("B") * 256 + ord("C")]], result)
-      else:
-        self.assertAllEqual(
-            [[ord("A") + ord("a") * 256, ord("B") + ord("C") * 256]], result)
+      self.assertAllEqual(
+          [[ord("A") + ord("a") * 256, ord("B") + ord("C") * 256]], result)
 
       with self.assertRaisesOpError(
           "Input to DecodeRaw has length 3 that is not a multiple of 2, the "
           "size of int16"):
         decode.eval(feed_dict={in_bytes: ["123", "456"]})
 
+  def testEndianness(self):
+    with self.test_session():
+      in_bytes = array_ops.placeholder(dtypes.string, shape=[None])
+      decode_le = parsing_ops.decode_raw(
+          in_bytes, out_type=dtypes.int32, little_endian=True)
+      decode_be = parsing_ops.decode_raw(
+          in_bytes, out_type=dtypes.int32, little_endian=False)
+      result = decode_le.eval(feed_dict={in_bytes: ["\x01\x02\x03\x04"]})
```

Comment:
Won't this depend on the endianness of the machine where this test is run? You can test the endianness of the machine here using the Python sys module, e.g.,
```
>>> import sys
>>> sys.byteorder
'little'
```